src/utils/return_periods.py

```diff
@@ -0,0 +1,123 @@
+import pandas as pd
+import numpy as np
+from scipy.stats import skew, norm, pearson3
+from lmoments3 import distr, lmom_ratios
+
+
+def lp3_params(x, est_method="lmoments"):
```

Comment:
Is there a use case for having these params as an intermediate output? Otherwise I wonder if it might make sense to just put this function inside `lp3_rp` so the user only has to specify the method.

Reply:
yeah for the purpose of the actual data set pipeline i think its fine to wrap them together. 

However, they are more useful separate for general analysis as `lp3_params()` is calculated on an aggregated data set where the block maxima is calculated over any desired time block. The `lp3_rp()` command then takes the those inputs and applies them to any random vector -- which for our purpose is the values from the "raw" data set (not aggregated).

You could combine them together in a more sophisticated wrapper where you specify the time-step for the first aggregation, but it seems like it might end up not as flexible as we often went in various analyses